jenkinsfiles/ginkgo-kernel.Jenkinsfile

```diff
@@ -126,7 +149,7 @@ pipeline {
                 unsuccessful {
                     script {
                         if  (!currentBuild.displayName.contains('fail')) {
-                            currentBuild.displayName = 'K8s 1.17 vm provisioning fail\n' + currentBuild.displayName
+                            currentBuild.displayName = 'K8s vm provisioning fail\n' + currentBuild.displayName
```

Comment:
You might need to put the \n after the `currentBuild.displayName` or did you mean to have this be on 2 lines (would it even fit as the name not the description of the build?)

Reply:
It's meant to have the \n there, formatting is a bit nicer with it.